README.md

```diff
@@ -1,9 +1,5 @@
 # Dynatrace OpenTelemetry Metrics Exporter for .NET
 
```

Comment:
I wonder if we should align with the other exporter text in here. For ex, in Java, we have a paragraph here + a paragraph now with which version of the OTel SDK this exports works it/targets. 

https://github.com/dynatrace-oss/opentelemetry-metric-java/pull/21/files

Reply:
That's a good idea. I like this :slightly_smiling_face: 
I took that text and adapted it for .NET in [2a78af1](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/31/commits/2a78af13846b505649ceffe1dbc9643cc5381c83).
Kudos to @pirgeo for writing it in the first place :bowing_man: